### ISSUE TITLE
Support for _score in PercolateMany

### ISSUE THREAD

**Issue by joehmchan** [elastic/elasticsearch-net#1570]
PercolateMany has TrackScores() method for tracking scores, but in PercolateResponse.cs, PercolatorMatch class doesn't have Score field.

Should it look like this?
```c#
public class PercolatorMatch
{
	[JsonProperty(PropertyName = "_index")]
	public string Index { get; set; }
	[JsonProperty(PropertyName = "_id")]
	public string Id { get; set; }
	[JsonProperty(PropertyName = "highlight")]
	public Dictionary<string, IList<string>> Highlight { get; set; }
	[JsonProperty(PropertyName = "_score")]
	public double? Score { get; set; }
}
```

**Comment by Mpdreamz** [comment]
Hey @joehmchan at first quick cursory glance I think you are right what happened with your PR?

**Comment by joehmchan** [comment]
I tried to percolate all doc in an index type by _mpercolate and find percolator matches with score. The score is for ranking, 

I was able to do so via elasticsearch RESTful Api, but NEST doesn't expose the Score field.

**Comment by joehmchan** [comment]
Hi @Mpdreamz, will this change be included to next build?

**Comment by Mpdreamz** [comment]
Hi @joehmchan it will be but our initial focus right now is NEST 2.0, we will get an bug fix release (1.7.2) out after that

**Comment by Mpdreamz** [comment]
This was fixed in https://github.com/elastic/elasticsearch-net/pull/1655 and should make it into the next release of the 1.x branch. Thanks for reporting and sorry for this taking a while to resolve :+1:
